<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#4A646C;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_judul">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>811</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Handwriting</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white; background:#4A646C;</string>
    </property>
    <property name="text">
     <string>CUSS LAUNDRY</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_admin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>ADMIN :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_admin">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background:#4A646C;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="lb_nama">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Nama :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_nama">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_hp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>No. HP :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_hp">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_alamat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Alamat :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_alamat">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_nota">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Nota :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_nota">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string>background:#4A646C;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="lb_paket">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Paket Laundry :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_berat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Berat (kg) :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_jumlah">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Jumlah :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_paket">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
     <item>
      <property name="text">
       <string>Cuci Kering</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cuci Basah</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cuci Setrika</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="sb_berat">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_jumlah">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_status">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Status :</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="lunas">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Lunas</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="belum_bayar">
     <property name="geometry">
      <rect>
       <x>201</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Belum</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_hitung">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white; background:#4A646C;</string>
    </property>
    <property name="text">
     <string>H I T U N G</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_total">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>T O T A L :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_tgl_masuk">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Tanggal Masuk</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_tgl_selesai">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Tanggal Selesai</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tambah">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>610</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white; background:#4A646C;</string>
    </property>
    <property name="text">
     <string>TAMBAH </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>750</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background:#4A646C;</string>
    </property>
    <property name="text">
     <string>SIMPAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>750</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background:#4A646C;</string>
    </property>
    <property name="text">
     <string>HAPUS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>750</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background:#4A646C;</string>
    </property>
    <property name="text">
     <string>CARI DATA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cari">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>610</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white; background:#4A646C;</string>
    </property>
    <property name="text">
     <string>CARI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_hapus">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>610</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white; background:#4A646C;</string>
    </property>
    <property name="text">
     <string>HAPUS</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_data_pelanggan">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Data Pelanggan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cetak_nota">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>610</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white; background:#4A646C;</string>
    </property>
    <property name="text">
     <string>CETAK NOTA</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>470</y>
      <width>681</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_db">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_tabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_total">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>660</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>312</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
